examples/test.avsc

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
This file is named test.
If this is part of the examples it should be renamed. Or else it should be removed

Reply:
Removed